src/TraceEvent/EventSources/ProcessMetadataEventSource.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.Diagnostics.Tracing
+{
+    [EventSource(Name = "ProcessMetadataEventSource")]
+    public sealed class ProcessMetadataEventSource : EventSource
+    {
+        public class Tasks
```

Comment:
I intentionally steered away from them, because it would seem like process metadata is a sort of monolith. I'm happy to add them now or later if you see it differently.